src/lib/utils/titleCase.ts

```
@@ -1,11 +1,13 @@
 export function toTitleCase(str: string): string {
     const minorWords = new Set(['a', 'an', 'the', 'and', 'but', 'or', 'for', 'nor', 'on', 'at', 'to', 'with', 'in']);
     
-    const romanNumeralPattern = /^(I{1,3}|IV|V|VI{1,3}|IX|X)$/i;
+    const romanNumeralPattern = /(^|\s|-)(I{1,3}|IV|V|VI{1,3}|IX|X)(\s|$|:)/i;
 
     return str.split(' ').map((word, index) => {
-        if (romanNumeralPattern.test(word)) {
-            return word.toUpperCase();
+        const romanMatch = word.match(romanNumeralPattern);
+        if (romanMatch) {
+            const [fullMatch, prefix, romanPart] = romanMatch;
```

ESLint

[error] 'fullMatch' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)